ui/components/editor/EditorSubmitDialog.vue

```diff
@@ -140,37 +261,37 @@ export default {
 
       return rawData;
     },
-    canDownload() {
-      // Hide the download button in case the fixture has errors. Some plugins are not able to export such fixtures.
-      return process.client && !this.hasValidationErrors;
+    hasPreview() {
+      if (this.fixtureCreateResult === null) {
+        return false; // undetermined
```

Comment:
Correct wording would be *indeterminate*, but I think you can just drop the comment.